Carthage/Checkouts/PINCache/Source/PINDiskCache.h

```diff
@@ -19,8 +19,8 @@ extern NSErrorUserInfoKey const PINDiskCacheErrorWriteFailureCodeKey;
 extern NSString * const PINDiskCachePrefix;
 
 typedef NS_ENUM(NSInteger, PINDiskCacheError) {
-  PINDiskCacheErrorReadFailure = -1000,
-  PINDiskCacheErrorWriteFailure = -1001,
+    PINDiskCacheErrorReadFailure = -1000,
+    PINDiskCacheErrorWriteFailure = -1001,
```

Comment:
I think you could revert this formatting change, too.

Reply:
i could but previously they're not right indented?

Reply:
It's fine then. It's just not related to the rest of this change.